server/sockets/index.js

```diff
@@ -92,15 +100,16 @@ const socketListenTo = server => {
     });
 
     addItemToListWS(socket);
+    addListMemberWS(socket, dashboardViewClients, cohortViewClients);
     archiveItemWS(socket);
     updateItemState(socket);
     deleteItemWS(socket);
     restoreItemWS(socket);
     updateItemWS(socket);
     addCommentWS(socket);
-    sendListsOnAddCohortMember(socket, dashboardClients);
+    sendListsOnAddCohortMemberWS(socket, dashboardViewClients);
 
-    addCohortMemberWS(socket, cohortsClients);
+    addCohortMemberWS(socket, allCohortsViewClients);
```

Comment:
why this empty line in 111?

Reply:
To separate list's and cohort's listeners